### PR TITLE
feat: add public notebook for evaluating models in Vertex AI Studio and model garden

### PR DESCRIPTION
**Pull request by jsondai**
# Description
feat: add public notebook for evaluating models in Vertex AI Studio and model garden
chore: update introduction texts and titles for gen ai evaluation service tutorials

### CODE REVIEW

File: gemini/evaluation/evaluate_models_in_vertex_ai_studio_and_model_garden.ipynb [hunk 1]
```diff
@@ -0,0 +1,2273 @@
+{
```
Comment (inardini) [inline gemini/evaluation/evaluate_models_in_vertex_ai_studio_and_model_garden.ipynb:1]: <p>Replace PROJECT_ID with the template placeholder. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/generative-ai/pull/1192/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>

Comment (inardini) [inline gemini/evaluation/evaluate_models_in_vertex_ai_studio_and_model_garden.ipynb:1]: <p>Adding some explanation of EvalTask here would be beneficial for practitioners. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/generative-ai/pull/1192/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='37'/>

**Comment by jsondai** [comment]
<p>Added a section titled: "Understand the <code>EvalTask</code> class"  </p>

**Review by inardini — APPROVED** [review]
Despite 14 file changes, the main change is the one which introduces the new `evaluate_models_in_vertex_ai_studio_and_model_garden.ipynb` notebook. 

LGTM

**Pull request merged**